homeassistant/components/motionmount/entity.py

```diff
@@ -80,6 +84,8 @@ async def _ensure_connected(self) -> bool:
 
         Returns false if the connection failed to be ensured.
         """
+        if TYPE_CHECKING:
+            assert self.platform.config_entry
```

Comment:
What even is `self.platform`?

Reply:
Glad you ask. By lack of documentation (or my inability to find it) I've browsed the source code and found this `platform` variable. It works like a charm. How do you suggest I get the corresponding `config_entry`?